test/testdata/workspaces/python-flask-sample-app/.coveragerc

```diff
@@ -0,0 +1,5 @@
+# .coveragerc to control coverage.py
+[run]
+branch = True
+omit = *_test*
+       *__init__*
```

Comment:
In general there are a few lines without EOF character (final newline). Could you configure your editor to always add them? That would be great!

Reply:
Sure, I've just added the following to my VSCode settings file:

```
  "files.insertFinalNewline": true,
  "files.trimFinalNewlines": true,
````

Reply:
Don't these contradict each other? Shouldn't `"files.trimFinalNewlines": true,` be `false`? :)